### PR TITLE
upgraded errai version

### PR DESCRIPTION
**Pull request by mbiarnes**

### CODE REVIEW

**Comment by mbiarnes** [comment]
https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/943
https://github.com/kiegroup/appformer/pull/673

File: pom.xml [hunk 1]
```diff
@@ -101,7 +101,7 @@
     <version.com.google.guava.for-selenium>23.0</version.com.google.guava.for-selenium>
 
     <version.com.google.elemental2>1.0.0-beta-1</version.com.google.elemental2>
-    <version.org.jboss.errai>4.6.0-SNAPSHOT</version.org.jboss.errai>
+    <version.org.jboss.errai>4.7.0-SNAPSHOT</version.org.jboss.errai>
```
Comment (mareknovotny) [inline pom.xml]: what about starting to use fixed version here
not the 4.7.0-SNAPSHOT but released 4.6.0.Final

**Comment by mbiarnes** [comment]
+1 - will change this later

File: pom.xml (hunk 1 above)
Comment (mbiarnes) [reply pom.xml]: fixed

**Pull request merged**